recipes/libevent/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2.0.22" %}
+
+package:
+  name: libevent
+  version: {{ version }}
+
+source:
+  fn: libevent-{{ version }}.tar.gz
+  url: https://github.com/libevent/libevent/archive/release-{{ version }}-stable.tar.gz
+  sha256: ab89639b0819befb1d8b293d52047c6955f8d1c9150c2b22a0e6247930eb9128
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
```

Comment:
Honestly not going to spend much time thinking about this. The reason being `libevent` is a library and `tmux` is a binary executable.

Neither really depend on Python. Though I did see a Python script being installed in `bin` by `libevent`. The script appears to be Python 2 only based on its shebang. Continuum appears to ship the script and not require Python.